.github/workflows/release-automation.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
         uses: "WyriHaximus/github-action-get-previous-tag@04e8485ecb6487243907e330d522ff60f02283ce" # v1.4.0
   generate-artifacts:
     needs: get-latest-tag
-    runs-on: ubuntu-latest
```

Comment:
Is this to fix the compat issue of the dynamic linked binaries?
How do you know this OS version is old enough?
Did you test the binary on the older OSes?

Reply:
Updated the ubuntu version to 20.04